Coordinates coordinates
CovarianceMatrix covariance_matrix
ConeColor[] cone_color
float32 cone_probability
